.vscode/tasks.json

```diff
@@ -87,7 +87,7 @@
         "${input:verboseOutput}",
         "-i",
         "${input:packagesToIgnore}",
-        "-r",
+        "-o",
```

Comment:
renamed release override value to "o" so that "r" could be used for the version bump type. This way it matches what we have in other repos.